### PR TITLE
@types/vast-client: add missing track() method to VASTTracker

### PR DESCRIPTION
**Pull request by yowcow**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/dailymotion/vast-client-js/blob/master/src/vast_tracker.js#L396
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/vast-client/vast-client-tests.ts [hunk 1]
```diff
@@ -322,3 +322,12 @@ vastTracker.on('clickthrough', (url: string) => {
     // Open the resolved clickThrough url
     document.location.href = url;
 });
+
+// Bind acceptInvitation listener to the invitation button
+const invitationButton = document.getElementById('inviationButtonId') as HTMLButtonElement;
```
Comment (jgainfort) [inline types/vast-client/vast-client-tests.ts]: little typo

inviation -> invitation

File: types/vast-client/index.d.ts [hunk 2]
```diff
@@ -128,6 +128,20 @@ export class VASTTracker extends EventEmitter {
...
+    /**
+     * Calls the tracking URLs for the given eventName and emits the event.
+     */
+    track(
+        /**
+         * The name of the event. Call the specified event tracking URLs. Emit the specified event when done.
+         */
+        eventName: string,
+        /**
+         * Indicate if the event has to be tracked onlye once.
```
Comment (jgainfort) [inline types/vast-client/index.d.ts]: typo

onlye -> only

**Review by jgainfort — CHANGES_REQUESTED** [review]
Just the two typos, otherwise it looks great. Thank you.

File: types/vast-client/vast-client-tests.ts (hunk 1 above)
Comment (yowcow) [reply types/vast-client/vast-client-tests.ts]: fixed, thanks!

File: types/vast-client/index.d.ts (hunk 2 above)
Comment (yowcow) [reply types/vast-client/index.d.ts]: fixed

**Review by jgainfort — APPROVED** [review]

**Pull request merged**